hack/jenkins/job-builder-image/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# This docker image runs Jenkins Job Builder for automatic job reconciliation.
+
+FROM ubuntu:14.04
```

Comment:
(Don't spend much time on this, it's not important for this to be optimized.  Just might be easier to maintain.  Or not.)